packages/design-system-react-native/package.json

```diff
@@ -43,35 +43,35 @@
     "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch"
   },
   "dependencies": {
-    "@metamask/design-system-twrnc-preset": "workspace:^",
-    "@metamask/design-tokens": "workspace:^",
-    "@metamask/utils": "^11.4.0",
-    "@solana/addresses": "^2.1.0",
-    "@testing-library/react-hooks": "^8.0.1",
-    "bitcoin-address-validation": "^3.0.0",
-    "react-native-gesture-handler": "^1.10.3",
-    "react-native-jazzicon": "^0.1.2",
-    "react-native-reanimated": "3.3.0"
+    "react-native-jazzicon": "^0.1.2"
```

Comment:
These dependencies have been moved to either peerDependencies or devDependencies based on their usage. Packages like testing-library are only needed for development and testing, so they are now devDependencies. The other packages are used by mobile outside of the design system’s UI components, so moving them to peerDependencies allows the consuming app to specify the version, reducing the overall bundle size and avoiding duplicate dependencies. We are keeping `react-native-jazzicon` as a regular dependency because it should only be used internally by AccountAvatar within design-system-react-native, not directly by consuming applications.